validations/assets.js

```diff
@@ -89,7 +89,8 @@ function validateSVGFiles(article){
  * @returns an array of ValidationIssue objects for the found issues.
  */
 function validateAssetsFolderName(article){
-    if(article.assets.length > 0 && article.assetsFolder === null){
+
+    if(article.assets.length > 0 && article.assetsFolder != "assets"){
```

Comment:
This changes the behavior. "images" should also be allowed. Those cases should be handled by the `assetsFolder` property. If it doesn't return null, the problem is elsewhere.

Reply:
Then on the assetsFolder function
// Try to figure out assets path from the referenced images
        const usedAssetPaths = this.referencedImages.map((assetPath) => {
            const directory = path.dirname(assetPath)
            if(!directory) return null;
            return directory.split("/")[0];
        })        ````

Reply:
`assetsFolder` just returns the folder, it doesnt matter the name, in the `assets.js` should be the logic which determines if it is correct or not, isnt it?

Reply:
@marqdevx You're right. Let me push a commit.